be/src/formats/parquet/group_reader.cpp

```diff
@@ -101,47 +91,80 @@ Status GroupReader::get_next(ChunkPtr* chunk, size_t* row_count) {
         has_filter = _dict_filter_ctx.filter_chunk(&active_chunk, &chunk_filter);
```

Comment:
one suspicious bug:
- we have set the filter to skip some row ids
- however, we use `pred->evaluate` inside `_dict_filter_ctx.filter_chunk`
- maybe change `pred->evaluate` to `pred->evaluate_and`